src/d3a/models/strategy/predefined_pv.py

```diff
@@ -115,8 +115,8 @@ class PVUserProfileStrategy(PVPredefinedStrategy):
     """
         Strategy responsible for reading a profile in the form of a dict of values.
     """
-    parameters = ('panel_count', 'risk', 'final_selling_rate', 'energy_rate_decrease_option',
-                  'energy_rate_decrease_per_update', 'power_profile', 'initial_selling_rate')
+    parameters = ('panel_count', 'final_selling_rate', 'energy_rate_decrease_per_update',
+                  'power_profile', 'initial_selling_rate')
```

Comment:
@spyrostz Would there be any harm if simply this class variable is removed? Or is there any advantage of having it?

Reply:
This needs to stay here, otherwise this would cause these parameters to not be read when sent by d3a-web. IIRC the area deserializer requires this 'parameter' class variable which contains serialized versions of the constructor arguments.